config/envs/default.py

```diff
@@ -64,3 +69,9 @@ class DefaultConfig:
     ASSETS_AUTO_BUILD = True
 
     TALISMAN_SETTINGS = CommonConfig.TALISMAN_SETTINGS
+
+    # TODO move into common config
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
```

Comment:
these were put into common config as part of the 1.0.10 release :)

Reply:
amazing! Will update now :)